variants/feather_m0/linker_scripts/gcc/flash_with_bootloader.ld

```diff
@@ -1,216 +0,0 @@
-/*
```

Comment:
We need these for each of MCCI's boards, and we don't want to remove the Adafruit ones.